src/components/RequestedPackageList.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import Accordion from "@mui/material/Accordion";
+import Typography from "@mui/material/Typography";
+import ArrowRightRoundedIcon from "@mui/icons-material/ArrowRightRounded";
+import RequestedPackage from "./RequestedPackage";
+import Box from "@mui/material/Box";
+import StyledAccordionSummary from "src/styles/StyledAccordionSummary";
+import StyledAccordionDetails from "src/styles/StyledAccordionDetails";
+import useTheme from "@mui/material/styles/useTheme";
+
+interface IRequestedPackageListProps {
+  packageList: (string | object)[];
+  listHeight: number;
+}
+
+const RequestedPackageList = ({
+  packageList,
+  listHeight,
+}: IRequestedPackageListProps) => {
+  const { palette } = useTheme();
+
+  const filteredPackageList = packageList.filter(
+    (item) => typeof item !== "object"
+  );
+  const listLength = filteredPackageList.length;
+
+  return (
+    <Accordion sx={{ width: 421, boxShadow: "none" }}>
+      <StyledAccordionSummary
+        expandIcon={
+          <ArrowRightRoundedIcon
+            sx={{ width: 51, height: 55, color: palette.secondary.main }}
+          />
+        }
+      >
+        <Typography
```

Comment:
`Typography` is fine for now, but when we consider the editable version we may want to switch this to an `Autocomplete` with `disabled=true`

Reply:
@telamonian  this `Typography` is for the title of the "Request packages" component. I guess your comment applies to the other `Typography` components used to represent the packages names, right ?

Reply:
@ostojics yes that's what i meant to refer to. that and the field for the package version